sdk/eventhub/event-hubs/README.md

```diff
@@ -223,42 +225,42 @@ main();
 Azure Event Hubs is capable of dealing with millions of events per second. 
 To scale your processing application, you can run multiple instances of your application and have it balance the load among themselves.
 
-Begin by creating an instance of the `EventHubConsumerClient`, and then call the `subscribe()` method on it to start
-consuming events. Pass an instance of a `PartitionManager` to the `subscribe()` method which the `EventHubConsumerClient` can 
-use to co-ordinate the processing between the multiple instances of your application.
+Begin by creating an instance of the `EventHubConsumerClient` using one of the
+constructor overloads that take a `CheckpointStore`, and then call the `subscribe()` 
+method to start consuming events. The checkpoint store will enable the subscribers 
+within a consumer group to coordinate the processing between multiple instances 
+of your application.
 
-In this example, we will use the `PartitionManger` from the `@azure/eventhubs-checkpointstore-blob` package
+In this example, we will use the `BlobCheckpointStore` from the `@azure/eventhubs-checkpointstore-blob` package
 which implements the required read/writes to a durable store by using Azure Blob Storage.
 
-The [subscribe](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.0-preview.6/classes/eventhubconsumerclient.html#subscribe) 
+The [subscribe](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.0-preview.7/classes/eventhubconsumerclient.html#subscribe) 
```

Comment:
Lets remove the link to `subscribe` here for the same reason as https://github.com/Azure/azure-sdk-for-js/pull/6353/files#r351551937